rust/azure_iot_operations_services/src/azure_device_registry.rs

```diff
@@ -173,38 +170,18 @@ impl From<discovery_client_gen::CodeSchema> for base_client_gen::CodeSchema {
     }
 }
 
-impl From<StatusConfig> for base_client_gen::DeviceStatusConfigSchema {
+impl From<StatusConfig> for base_client_gen::ConfigStatus {
```

Comment:
From ConfigStatus to StatusConfig. Interesting.
I was going to ask if StatusConfig is a good name, but I think this is what Carl was sharing about earlier today.

Reply:
yeah, it's interesting - we originally had StatusConfig to match their DeviceStatusConfigSchema, but then they swapped it...not sure if we should take this swap or not - might be clearer? I'll make the change - never bad to be more consisten with the service

Reply:
Realized as well that I refer to this as the config status in all of my doc, so that's an extra good reason to make the change. Updated